### ISSUE TITLE
[bug] Theme Switching Issues

### ISSUE THREAD

**Issue by radeeyate** [micahlt/wasteof.mobile#36]
I've noticed that when switching system theme doesn't fully update all factors of themes, specifically navigation bar icons. The theming will fix when you either switch users or restart the app.
![Screenshot_20230613-124509](https://github.com/micahlt/wasteof.mobile/assets/124477460/e247014a-8c12-4794-92d8-42655223e759)
![Screenshot_20230613-124348](https://github.com/micahlt/wasteof.mobile/assets/124477460/9391623e-8b76-4ad1-bb23-b19cabb102b5)

**Comment by micahlt** [comment]
Yes, this a known (if undocumented) bug.  It involves a problem with `react-navigation` and is a very niche/minimal issue, so bugfixes will be accepted but this issue will be placed in the backlog for now.
